Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -38,6 +38,48 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Name of the artifact to publish. If not set, defaults to a unique ID scoped to the job."
+        }, 
+        {
+            "name": "artifactType",
+            "aliases": [
+                "publishLocation"
+            ],
```

Comment:
why exactly do we need an alias here? Can you show the previous implementation so I get an idea of what it needs to be?

Reply:
This is just for consistency with the build artifact task. Some ppl who are using yaml might already be using the name publishLocation for a while. I'll leave it to Will regarding if we want to update the name to something else.

Reply:
I suggested (below) just having a simple boolean input that lets the [small number of advanced users] opt into publishing to a file share. The default is publishing as usual (as an artifact).